README.md

```
@@ -333,17 +336,27 @@
+
+- **`mcp-subdomain`** - `service.subdomain.conf` + MCP features → Streamable-HTTP/SSE support
+- **`mcp-subfolder`** - `service.subfolder.conf` + MCP features → Path-based with SSE support
+
+#### Template Selection
+
+- Base type inferred from filename: `.subdomain.conf` or `.subfolder.conf`
+- MCP features added when `mcp_enabled=true` parameter is used
+- No need to specify template type separately - it's determined automatically
 
-**MCP Template Features:**
+#### MCP Template Features
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)